app/js/lib/foxbox/foxbox.js

```diff
@@ -355,7 +355,7 @@ export default class Foxbox extends Service {
     return this[p.net].fetchJSON(`${this[p.net].origin}/services/${id}/state`)
       .then(res => {
         if (!res) {
-          throw new Error(`The action couldn't be performed.`);
+          throw new Error('The action couldn\'t be performed.');
```

Comment:
> But I understand your point so let's stick with single quote and back tick only if there's a parameter.

Sounds good! We can return back to this style later if we feel we need this change :)